clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackImageProvider.groovy

```diff
@@ -16,6 +16,7 @@
 
 package com.netflix.spinnaker.clouddriver.openstack.client
 
+import com.netflix.spinnaker.clouddriver.openstack.model.OpenstackImage
 import org.openstack4j.model.image.Image
```

Comment:
I think this import of `org.openstack4j.model.image.Image` is now unused for the interface.

Reply:
Done